hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanCancelFlowTask.groovy

```diff
@@ -0,0 +1,203 @@
+      responseList.each { execResponse ->
+        JSONObject execIDResponse = new JSONObject(execResponse);
+        if (execIDResponse.has("error")) {
+          logger.error("Could not get flow due to ${execIDResponse.get("error").toString()}");
+        } else if (execIDResponse.has("execIds")) {
+          JSONArray execArray = execIDResponse.getJSONArray("execIds");
+          if (execArray != null) {
+            for (int i=0; i<execArray.length(); i++) {
+              execIds.add(execArray.get(i).toString());
+            }
+          }
+        }
+      }
+
+      if (!execIds.isEmpty()) {
+        Set<String> runningExecSet;
+
+        if (AzkabanPlugin.interactive) {
+          //Pool HTTP Get requests for getting exec Job statuses
+          responseList = AzkabanClient.batchfetchFlowExecution(azkProject.azkabanUrl, execIds, sessionId);
+          printStatus(flows, responseList);
+
+          //Get exec ID's to kill
+          String input =  AzkabanHelper.consoleInput(System.console(), " > Enter IDs of executions to be killed:", false);
+          if(input.isEmpty()) {
+            logger.error("Entered Nothing, Exiting...");
+            return;
+          }
+
+          runningExecSet = new HashSet<String>(Arrays.asList(input.split("\\D+")));
+          if (runningExecSet != null && !runningExecSet.isEmpty()) {
+            runningExecSet.sort();
+            //Check if entered execIds are from this project
+            for (String runningExec: runningExecSet) {
+              if (!execIds.contains(runningExec)) {
+                logger.error("Enter execution IDs from current project");
+                return;
+              }
+            }
+          } else {
+            logger.error("Empty indices");
+            return;
+          }
+        } else {
+          List<String> execs = project.getProperties().get("execId").toString().split(",+");
+          runningExecSet = new HashSet<String>();
+
+          for (String exec : execs) {
+            runningExecSet.add(exec);
+          }
+        }
+
+        if (!runningExecSet.isEmpty()) {
+          //Pool HTTP Get requests for killing flows
+          responseList = AzkabanClient.batchCancelFlowExecution(azkProject.azkabanUrl, runningExecSet, sessionId);
+
+          for (String killResponse : responseList) {
+            JSONObject killResponseObj = new JSONObject(killResponse);
+            if (killResponseObj.has("error")) {
+              logger.error(killResponseObj.get("error").toString());
+            } else {
+              logger.lifecycle("Successfully killed flow execution");
+            }
+          }
+        } else {
+          logger.lifecycle("Entered no execution Ids");
+        }
+
+      } else {
+        logger.lifecycle("\nProject " + azkProject.azkabanProjName + " has no running flows to kill.");
+      }
+
+    } catch (IOException ex) {
+      ex.printStackTrace();
+    }
+  }
+
+  void printStatus(List<String> flows, List<String> responseList) {
```

Comment:
Can we move the printStatus method to an AzkabanConsoleClient helper class in the azkaban-cli project?